chains/orchestrator-relays/runtime/dancelight/src/tests/integration_test.rs

```diff
@@ -356,6 +358,197 @@ fn test_container_deregister_unassign_data_preserver() {
         });
 }
 
+#[test]
+fn stream_payment_works() {
```

Comment:
Same for tests, we need to find a way reusing tests for different nodes, probably in the another PR.

Reply:
Created a ticket for common code (if possible)